src/v/v8_engine/api.cc

```diff
@@ -27,10 +27,24 @@ ss::future<> executor_service::stop() { return _executor.stop(); }
 api::api(ss::alien::instance& instance, unsigned core, int64_t size)
   : _executor_service(instance, core, size) {}
 
-ss::future<> api::start() { return _executor_service.start(); }
+ss::future<> api::start() {
+    co_await _executor_service.start();
+    co_await _dp_code_database.start();
+}
 
-ss::future<> api::stop() { return _executor_service.stop(); }
+ss::future<> api::stop() {
+    co_await _dp_code_database.stop();
```

Comment:
if the executor is on a single core then why is the code database on all cores?

Reply:
executor - is wrapper. It can be on one core or on each core. I do not want design all data_policy stuff for executor which run on only one core.
Also executor does not know anything about task inside, so if we put code_database on one core all task from different cores must get iobuf from this one core...